content/hardware/ordering.md

```diff
@@ -0,0 +1,101 @@
+
+**Processor**
+
+The MuSHR Racecar can be built with a choice of two main processors, the Nvidia Jetson Nano or the Nvidia Jetson Xavier NX. The latter is better for high-performance applications, while the former is better if you're working on a budget. You can view a comparison of the two [here.](https://www.seeedstudio.com/blog/2020/06/04/nvidia-jetson-nano-and-jetson-xavier-nx-comparison-specifications-benchmarking-container-demos-and-custom-model-inference/)
+
+**Sensing**
+
+MuSHR supports the Intel RealSense D435/D435i and the D455 cameras for depth perception. The car also has a scanning YDLIDAR X4, and other optional add-on sensors listed below. View a comparison of the different Intel RealSense cameras [here.](https://www.intelrealsense.com/compare-depth-cameras/)
+
+**Chassis**
+
+The base chassis of the car is a Redcat Racing Rally Car with two motor options, modified extensively with 3D printed parts. The Pro chassis is more expensive, but follows a simpler build process, not requiring motor replacement, saving hours of build time.
+
+___
+## Add-on Modules 
+
+You can also pick add-on modules to supplement your car and augment its capabilities to perform certain functions.
+
+**Push Button**
+
+A button mounted to the front of the car, allowing you to detect if the front bumper is in contact with an object or surface.
+
+**T265 Tracking Camera**
+
+The MuSHR Racecar can have a second camera mounted to it, the Intel RealSense T265. This camera is particularly useful for tracking.
+
+**Radio FPV**
+
+If you need to tele-operate outdoors or over a longer distance outside standard Bluetooth/WiFi range, you can follow the steps in [this tutorial](/tutorials/radio_fpv) to add a radio-based teleoperation system to your racecar.
+
+**Pushing Bumper**
+
+You can mount a 3D-Printed bumper to the front of the car to perform object manipulation tasks with the car.
+
+___
+## Recommended Tier System
+
+As a result of the various options available for MuSHR, we have simplified the MuSHR Racecar into three different tiers, illustrated in the table below. 
+
+|Tier|MuSHR Lite|MuSHR|MuSHR Pro|
+|---|---|---|---|
+|Processor|Nano|Nano|Xavier NX|
+|Camera|None|D435|D455|
+|Chassis|Base|Base|Pro|
+|Cost(in US$)|750|1000|1600|
+
+<br></br>
+The MuSHR Lite is ideal for users who are working on a budget, like hobbyists, and the MuSHR Pro is ideal for researchers and academic institutions, who need higher performance and are willing to splurge. The MuSHR tier meets in the middle, with full sensing capabilities but lower performance than the Pro option. If you feel like you don't fit into any of these three categories, don't worry! We have you covered in the next section!
+
+View the BoM for each tier [here.](https://docs.google.com/spreadsheets/d/1VV5fGFJIEthfwishO181WcWzMAkPAFYT/edit#gid=598654221)
```

Comment:
I only see add on modules in mushr-lite. Don't we want a separate tab for that?

Reply:
I did have it repeated on each page, but the background coloring was only on the lite page. Probably makes more sense to have it be on its own page so I've updated the BoM accordingly.